config/webpack/test.js

```diff
@@ -1,3 +1,7 @@
+/* global process, module, require */
+
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
```

Comment:
Shouldn't this be `'test'`?

```suggestion
process.env.NODE_ENV = process.env.NODE_ENV || 'test';
```

Reply:
@nickytonline maybe? https://github.com/rails/webpacker/blob/master/lib/install/config/webpack/test.js

🤷

Reply:
I asked a question, https://github.com/rails/webpacker/issues/2602, in their repo about this.

Reply:
I have the feeling the reason why it's working is that when you run webpacker it sets `NODE_ENV` correctly, so it never gets to the `OR` second clause